README.md

```diff
@@ -1185,6 +1185,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Motor Vehicle Department, Maharashtra, India](https://apisetu.gov.in/public/api/transportmh) | Driving License (DL) and Vehicle Registration Certificate (RC) of the State Maharashtra of India | `apiKey` | Yes | No |
 | [NHTSA](https://vpic.nhtsa.dot.gov/api/) | NHTSA Product Information Catalog and Vehicle Listing | No | Yes | Unknown |
 | [Smartcar](https://smartcar.com/docs/) | Lock and unlock vehicles and get data like odometer reading and location. Works on most new cars | `OAuth` | Yes | Yes |
+| [Forza-API](https://forza-api.tk) | Show random image of car from Forza | No | Yes | Unknown |
```

Comment:
Remove the `API` from the title and change the link to this one: https://images.forza-api.tk/

```suggestion
| [Forza](https://images.forza-api.tk) | Show random image of car from Forza | No | Yes | Unknown |
```

Reply:
Ok, I will do it.